about.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/styles.css">
+    <title>Web 3.0 About</title>
+</head>
+<body>
+    <header class="header1">
+        <div class="header1-content">
+            <ul>
+                <li>English</li>
+                <li>My Page</li>
+                <li>Logout</li>
+           </ul>
+        </div>
+    </header>
+    <img src="img/close.png" alt="close button" class="close-btn">
+    <header class="desktop-header">
+        <nav class="navbar" id="navbar-list">
+            <ul>
+                <li id="logo"><a href="index.html" rel="noreferrer noopener"><img src="img/logo.png" class="logo-image" alt="logo"></a>
+                </li>
+                <li><a href="index.html" rel="noreferrer noopener">Home</a></li>
+                <li><a href="#" rel="noreferrer noopener">Program</a></li>
+                <li><a href="#" rel="noreferrer noopener">Join</a></li>
+                <li><a href="#" rel="noreferrer noopener">Sponsor</a></li>
+                <li><a href="#" rel="noreferrer noopener">News</a></li>
+                <li id="web-conf"><a href="#" rel="noreferrer noopener">Register Now!</a></li>
+            </ul>
+        </nav>
+    </header>
+    <section class="about" id="about1">
+        <img src="img/menu.png" width="40" height="40" alt="hamburger button" class="hamburger-button"
+            id="hamburger-btn2">
+        <div class="about-main">
+            <h2>"Hello, Decentralized World!"</h2>
+            <h3>Web Evolution<br>Web 3.0 Conference 2022</h3>
+        </div>
+        <div class="about-description">
+            <article>Web 3.0 Conference is a Tech Circus event. We are an independent company that produces educational events for the
+            community of digital design, product & UX. We collaborate with global clients, agencies and organisations to bring
```

Comment:
- [ ] **Required:** Kindly fix this section, the texts are hidden.

![image](https://user-images.githubusercontent.com/27709832/163115817-39d4cf01-e106-404e-b847-81a0faefb4db.png)

- [ ] **Required:** You have carefully selected your color scheme and Kudos for that. However, this task is aimed at making a given design by a client.
- Please stick to the given colors
![image](https://user-images.githubusercontent.com/27709832/163116816-b744064c-c8f0-417b-b481-b098ca75ae59.png)

- You have not used the font family in the title
![image](https://user-images.githubusercontent.com/27709832/163116719-2624ddd0-5ebf-479e-b453-49e526d366b9.png)

Reply:
When I use cocogoose font family, the numbers are not displayed because for the pro-trial font, numbers are not available. That's why I used Lato for everything

<img width="384" alt="Screenshot 2022-04-14 at 03 49 36" src="https://user-images.githubusercontent.com/98121137/163292715-7995b3af-db4c-463d-9f59-d76f9ead3fb9.png">